driver-sync/src/test/functional/com/mongodb/client/ClientEncryptionCustomEndpointTest.java

```diff
@@ -237,6 +240,19 @@ public void testCustomEndpoint() {
                         + "  \"endpoint\": \"example.com:443\"\n"
                         + "}"),
                 false, MongoClientException.class, MongoCryptException.class, "Invalid KMS response"});
+        data.add(new Object[]{"10. [kmip] default endpoint",
+                "kmip",
+                BsonDocument.parse("{\n"
+                        + "  \"keyId\": \"1\"\n"
+                        + "}"),
+                false, null, null, null});
```

Comment:
```suggestion
                true, /* TODO */, /* TODO */, "TODO"});
```

The current proposed tests also check with an invalid endpoint.

`setUp` will need to add "kmip" to `invalidKmsProviders`:
```
put("kmip",  new HashMap<String, Object>() {{
    put("endpoint", "doesnotexist.local:5698");
}});
```

Reply:
Done